docs/make.jl

```diff
@@ -7,6 +7,9 @@ end
 
 using MuladdMacro
 
+cp("./docs/Manifest.toml", "./docs/src/assets/Manifest.toml", force = true)
+cp("./docs/Project.toml", "./docs/src/assets/Project.toml", force = true)
+
```

Comment:
It seems this relies on the fact that one runs the script from the root directory of the repository but is broken if this is not the case (e.g., locally I typically tend to build and preview docs by running the scripts from within the docs directory)? Maybe
```suggestion
cp(joinpath(@__DIR__, "Manifest.toml"), joinpath(@__DIR__, "src/assets/Manifest.toml"); force = true)
cp(joinpath(@__DIR__, "Project.toml"), joinpath(@__DIR__, "src/assets/Project.toml"); force = true)

```
would fix it.

Reply:
@devmotion Ah I understand. Just to confirm though, when the docs build through CI, is the repo setup to enter the docs directory and then run the build command? Most other Sciml repos that I have updated build the docs from the root directory, so I was anticipating the same thing. I can approve the adjustment you suggested.

Reply:
It completely depends on how the Github actions are implemented (I assume in most cases Github actions are used for building the docs, but IIRC some more compute intensive setups such as maybe SciMLSensitivity maybe they are run with buildkite?). With `@__DIR__` it should work regardless of how CI is configured or changed in the future.

Reply:
We should get a better solution for the paths here.